Ops/Ops-lessons-2022/CircleCI.md

```diff
@@ -28,6 +38,34 @@
 
 ### Orbs
 
+Orbs are reusable snippets of code that help automate repeated processes, accelerate project setup, and make it easy to integrate with third-party tools such as Node, Docker, and AWS. 
+
+To invoke an orb:
+
+```yaml
+  version: 2.1
+
+  orb:
+    maven: circleci/maven@0.0.12
+  
+  workflows:
+    maven_test:
+      jobs:
+        - maven/test
+```
+
+In this case, ``` maven: circleci/maven@0.0.12 ```, maven can be thought as the object reference variable and is used in the workflow named ``` maven_test ``` 
+
+You can create your own orb and use them within your organization only or post it on the circleci orb registry.
+
+SecurEd will be creating their own orb to make our config.yml shorters and abstract many jobs and commands to a single repository.
```

Comment:
Not for this PR but when we make this we should add a link to this playbook to the repo.

Reply:
Agreed